.devcontainer/README.md

```diff
@@ -0,0 +1,39 @@
+# Devcontainers
+
+Use [Devcontainers](https://code.visualstudio.com/docs/devcontainers/containers) to prepare a fully automated working environment.
+
+Generate the `devcontainer.json` executing:
+
+```shell
+.devcontainer/generate_devcontainer.sh
+```
+
+Now you should see the file `.devcontainer/devcontainer.json`. At this point you can use your favorite IDE to run Devcontainers
+
+## VSCode
+
+Install the extension https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers
+
+### Docker
+
+Docker defaults should work fine therefore there is nothing to do.
+
+### Podman
+
+Start Podman service for a regular user (rootless) and make it listen to a socket:
+
+```shell
+systemctl --user enable --now podman.socket
+```
+
+Restart your OS if necessary and verify that podman listens:
+
+```shell
+systemctl --user status podman.socket
+```
+
+Go to the Extension Settings:
+
+- `Dev › Containers: Docker Compose Path` set `podman-compose`
+- `Dev › Containers: Docker Path` set `podman`
+- `Dev › Containers: Docker Socket Path` set `/run/podman/podman.sock`
```

Comment:
Can you please add a step on running a devcontainer in vscode like "ctrl+shift+p" to open command palette and enter "Dev Containers: Rebuild and Reopen in Container"

Reply:
Added, thanks